src/Razor/test/RazorLanguage.Test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -471,7 +471,7 @@ public class Person
             var generated = CompileToCSharp(@"
 <InputText @bind-Value=""person.Name"" />
 
-@functions 
+@functions
```

Comment:
VS did this. I'm not sure why - but it did. Some of our tests had trailing whitespace and now they don't. There's some trivial diffs in the output as a result.

Reply:
That's weird. Can you close VS and run tests with GenerateBaselines=True from the command line and see if they match? Random whitespace diffs in baselines scare me.

Reply:
There no issue in the baselines. I'm saying VS changed the source code of the tests.

Reply:
As you can see here a space was removed.

Reply:
Ah totally misunderstood that. This is fine.